cmd/livepeer_cli/wizard_transcoder.go

```diff
@@ -268,3 +275,7 @@ func (w *wizard) showVoteChoices() {
 	}
 	wtr.Flush()
 }
+
+func feeShareToFeeCutString(feeShare *big.Int) string {
```

Comment:
What do you think about changing this something like `flipPerc()` and returning the `big.Int` representation 100% - X% where X is the argument to the function? Then, this function could be used anywhere where the `big.Int` representation of the fee cut is required and we could still convert that value into a float64 or string if needed as opposed to right now we'd go from big.Int -> big.Int -> string -> big.Int/float64 if we ever needed the value in a different format than a string.

For example:

```golang
...
} else {
    blockRewardCut = eth.ToPerc(orch.RewardCut)
    feeCut = eth.ToPerc(flipPerc(orch.FeeShare))
}

...

feeCut = readDefaultFloat(feeCut)

...

return blockRewardCut, HUNDRED_PERCENT - feeCut, pricePerUnit, pixelsPerUnit, serviceURI
```

And then anywhere we need to print the fee cut we could use use `eth.FormatPerc(flipPerc(orch.FeeShare))`